contributions/catalog/webapps/static/css/style.css

```diff
@@ -386,9 +386,33 @@ hr.dotted {
 
 #togglePassword {
     cursor: pointer;
-    position: absolute;
-    margin-left: -20px;
-    margin-top: -20px;
+    position: relative;
+}
+
+input[id^='environmentVariables_key'] {
```

Comment:
When applying the styles, reading CSS classes will be quicker than searching based on IDs. So, it might be better to move to these individual CSS classes, which are then used in the HTML code. Thanks.

Reply:
updated to classes.

Reply:
Thanks. Resolving this conversation.